x/inflation/module.go

```diff
@@ -6,6 +6,8 @@ import (
 	"fmt"
 	"math/rand"
 
+	"github.com/evmos/evmos/v10/x/inflation/exported"
```

Comment:
Q: Why is this structured differently compared to Ethermint (e.g. evmos/ethermint#1509)?

Reply:
@MalteHerrmann Good catch. I thought I had deleted the old files not sure why they are still on this PR.